test/python/sdpa/frost/test_sdpa_fwd_dsl_sm120.py

```
@@ -427,11 +456,17 @@ def _run_thd_case(
     for n in seq_q_lens:
         cu.append(cu[-1] + n)
     packed_o = o_storage[: cu[-1] * h_q * d_v].view(max(cu[-1], 1), h_q, d_v)
+    if check_stats and stats_layout == "head_major":
+        packed_stats = stats_storage.view(h_q, t_cap)  # (H, head_stride); tokens at [:, cu[i]:cu[i+1]]
+    elif check_stats:
+        packed_stats = stats_storage[: cu[-1] * h_q].view(max(cu[-1], 1), h_q)  # (T, H)
+    else:
+        packed_stats = None
     for i, (nq, nkv) in enumerate(zip(seq_q_lens, seq_kv_lens)):
```

Ruff (0.16.1)

[warning] Loop control variable `nkv` not used within loop body

Rename unused `nkv` to `_nkv`

(B007)